frontend/src/components/PrivateChat/PrivateChat.js

```diff
@@ -45,15 +45,12 @@ const PrivateChat = () => {
             data.allmsgg.forEach((m) => {
                 if (m.name === data.reciever) allmsg.push(m);
             });
-
-        // eslint-disable-next-line
```

Comment:
Please don't delete this line,
this line(inside **useEffect**) helps to clear warnings

```
// eslint-disable-next-line
```